aws-logging.yaml

```diff
@@ -0,0 +1,44 @@
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+data:
+  # NEVER set this to true, or set only temporary
+  # it enables internal fluentbit logs that flood Cloudwatch
+  # and cause huge AWS bills 
+  # (see slack post https://this-is-biomage.slack.com/archives/C014YMUT6GN/p1646915799716329)
+  flb_log_cw: "false"
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch
+        Match *
+        region eu-west-1
+        log_group_name fluent-bit-cloudwatch
+        log_stream_name $(kubernetes['pod_name'])
+        auto_create_group true
+
+  parsers.conf: |
+    [PARSER]
+        Name         docker
+        Format       json
+        Time_Key     time
+        Time_Format  %Y-%m-%dT%H:%M:%S.%L
+        Time_Keep    On
+  
+  filters.conf: |
+    [FILTER]
+        Name kubernetes
+        Match *
+        Merge_Log On
+        Buffer_Size 0
+        Kube_Meta_Cache_TTL 300s
+        Labels On
+        K8S-Logging.Exclude On
```

Comment:
This adds to the logs the kubernetes pod details, so that later on we can add pod name as log_stream_name. This will cause a separate stream to get created for each pod and sending the logs to it.
Documentation on how to configure this is in here: https://docs.fluentbit.io/manual/pipeline/filters/kubernetes